Add configuration details for APCU (#6076)
magento/devdocs: guides/v2.2/performance-best-practices/deployment-flow.md

bin/magento setup:di:compile
```

> After compilation completes, we recommend running the following command (you should also ensure [APCU is enabled]({{ page.baseurl }}/performance-best-practices/software.html#php-settings)):

Reviewer: ```suggestion
After compilation completes, confirm that [APCU is enabled]({{ page.baseurl }}/performance-best-practices/software.html#php-settings)), then run the following command:
```

Reviewer: Should the reference here be APCu as in other places?
